share/dmenu_run_hlwm

```diff
@@ -5,12 +5,18 @@ if ! command -v dmenu > /dev/null 2>/dev/null ; then
 	exit 1
 fi
 
+font=$(herbstclient attr theme.title_font)
```

Comment:
I don't think it is a very stable thing to take the font from hlwm's theme, because the font description might be incompatible easily. E.g. hlwm supports xft fonts, whereas dmenu does not support xft out of the box iirc.

Reply:
Actually it supports _only_ xft fonts.
Would it be sensible to use the font name from hlwm only if it does not look like a XLFD font description?

Reply:
ok. I now did just that. The font is ignored if it matches the pattern of a x font description.

Reply:
One can still do `spawn dmenu_run_hlwm -fn $font` in such a case.

Reply:
Sorry for not reacting!
You're right, I wasn't aware that dmenu now only supports xft. Thanks for incorporating the change!